cmd/kuberhealthy/util.go

```diff
@@ -52,10 +52,12 @@ func notifyChanLimiter(maxSpeed time.Duration, inChan chan struct{}, outChan cha
 		log.Println("channel notify limiter witnessed an upstream message on inChan")
 		for {
 			select {
-			case <-time.After(maxSpeed):
-				outChan <- struct{}{}
+			//case <-time.After(maxSpeed):
+			//	outChan <- struct{}{}
```

Comment:
commented this case out as outChan is notified every 10 seconds --> signaling the `externalChecksUpdateChanLimited` --> which reloads and restarts khcheck configurations